### DISCUSSION TITLE
Searching partial words

### DISCUSSION THREAD

**Discussion by nicolesherman00** [microsoft-search/pnp-modern-search#2367] [Q&A]
Hello!
I am trying to resolve a complaint we got for being able to search partial words?
Is there anyway to add a wildcard to the beginning and end of the search query so that partial words work without user having to manually enter the wildcard symbol?

For example, Searching for "Inform" in the searchbox yields no results. But if you type the whole word out "Information" you get a bunch of results. 

I saw in another post that beginning wild cards do not work... but least I can do is add it to the end if its possible.

Would love some help with this one. Thank you!

**Comment by wobba** [comment]
Wildcard at the end is supported, and if users tend to search for one term you can use this in the query template.

`{searchTerms} OR ({searchTerms}*)`

If you search for `foo` it will expand to `foo OR (foo*)` If you search for `foo bar` only the last term will get the wildcard `foo bar OR (foo bar*)`

Not 100%, but will likely increase satisfaction for your scenario.

**Reply by nicolesherman00** [reply]
Hi there, question about this.. When I add ({searchTerms}*) to the Query Template, it removes the document summary? 

![image](https://user-images.githubusercontent.com/22782498/212147355-846a90a7-71fe-4478-9bd6-9b517d07ca09.png)

![image](https://user-images.githubusercontent.com/22782498/212147521-f482aa81-09ce-483c-8353-de20034cec7b.png)

**Reply by wobba** [reply]
I cannot repro and get summaries for both.

**Reply by gretchunkim** [reply]
@nicolesherman00 Document summary can be added back in by modifying "Selected Fields" section.

**Reply by nicolesherman00** [reply]
It does not get removed.... There is nowhere to add it back. 
I dont specifically know where the selected fields section is, but I assume it is one of these and you can see the hithighlightedsummary is still selected in both.

![image](https://user-images.githubusercontent.com/22782498/213036096-0213a72a-2d8a-4e7c-ae4f-4e70a812562f.png)

![image](https://user-images.githubusercontent.com/22782498/213036162-d17f53f7-e12f-4e06-9ce2-6a3b1c3ccd12.png)

**Reply by gretchunkim** [reply]
@nicolesherman00 Make a duplicate of that specific webpart because you'll lose layout once you start changing these configuration settings. 

Now, one of them select "Debugging" layout. Then go through the entire list  to find out what that specific summary is called. You need to use that exact field name .If it doesn't show, you may need to type it in the Selected Properties. Once you have it showing up in the debug mode then you can go back to Layout slot to select that name.
